crates/walrus-service/src/node/storage/blob_info.rs

```diff
@@ -149,6 +161,120 @@ impl BlobInfoTable {
                 [(object_id, per_object_operation.to_bytes())],
             )?;
         }
+
+        batch.insert_batch(&latest_handled_event_index, [(&(), event_index)])?;
+        batch.write()
+    }
+
+    /// Updates the blob info for a blob based on the [`BlobEvent`] when the node is in recovery
+    /// with incomplete history.
+    ///
+    /// Only updates the info if the provided `event_index` hasn't been processed yet.
+    #[tracing::instrument(skip(self))]
+    pub fn update_blob_info_during_recovery_with_incomplete_history(
+        &self,
+        event_index: u64,
+        event: &BlobEvent,
+        first_epoch: Epoch,
```

Comment:
Nit: not used?

Reply:
Indeed. Weird that Clippy didn't complain about this... 🤔